linux_os/guide/services/ssh/ssh_server/sshd_use_strong_rng/rule.yml

```diff
@@ -0,0 +1,37 @@
+documentation_complete: true
+
+# TODO: The plan is not to need this for RHEL>=8.4
+prodtype: rhel8
+
+title: 'SSH uses strong entropy to seed'
+
+description: |-
+    Harden SSH by configuring it to use a seed from high-quality entropy source.
+
+rationale: |-
+    SSH implementation in RHEL8 uses the openssl library, which doesn't use high-entropy sources by default.
+
+severity: medium
+
+identifiers:
+    cce@rhel8: 82462-3
+
+references:
+    ospp: FIA_AFL.1
+
+ocil: |-
+    To determine whether SSH is configured to use strong entropy seed, run
+    <pre>$ sudo grep SSH_USE_STRONG_RNG /etc/sysconfig/sshd</pre>
+    If a line indicating that SSH_USE_STRONG_RNG is set to 1 is returned,
+    then the option is set correctly.
+
+ocil_clause: |-
+    The SSH_USE_STRONG_RNG is not set to 1 in /etc/sysconfig/sshd
+
+template:
+    name: shell_lineinfile
+    vars:
+        path: '/etc/sysconfig/sshd'
+        parameter: 'SSH_USE_STRONG_RNG'
+        value: '1'
+        no_quotes: 'true'
```

Comment:
The generated Ansible remediation contains an empty line with leading whitespace between `insertbefore` and `validate`.